src/admin-views/settings/emails/email-template.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * Tickets Emails Email Template
+ *
+ * @since  TBD  Email template to be mailed out by the Event Tickets plugin.
+ * 
+ * @var Tribe_Template  $this  Current template object.
+ */
+
+
+if ( tribe_is_truthy( $preview ) ) {
+	$this->template( 'email-template/preview' );
+} else {
+	$this->template( 'email-template/main' );
+}
```

Comment:
Once we are closer to release we should probably rely on the same template, receiving all the vars so that we don't have duplicated code, and so that's clear for the users making use of the template (overriding)

Reply:
There is no duplicate code. The `preview` template wraps the email template in a div to be used anywhere on a website. The `main` template adds the HTML header and meta info which is needed only for email clients. They both pull in the `style` and `body` templates, but in a different manner.

Reply:
Thanks for the clarification. What we should probably be doing here is including the templates and early bailing on each one of them by checking if it's preview or not. That way we're in line with the rest of templates we have all over the place.